client/src/components/Body.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+class Body extends Component {
+  render() {
+    return (
+      <main className="homePage">
+        <h1 className="pageTitle">Class2 Project</h1>
+        <img
+          className="image"
+          src="//image.shutterstock.com/image-vector/code-bee-logo-design-template-450w-721374241.jpg"
+          alt="code bee logo design template"
+          title="code bee logo design template"
+        />
+        <p className="paragraph">
```

Comment:
I think we could also ass a h2 tag for "About the Team" here followed with the paragraph with its description